MoEPluginBase/MoEPluginController.swift

```diff
@@ -0,0 +1,296 @@
+//
+//  MoEPluginController.swift
+//  MoEPluginBase
+//
+//  Created by Rakshitha on 23/06/22.
+//
+
+import Foundation
+import MoEngageSDK
+import MoEngageInApps
+import UserNotifications
+import CoreText
+
+class MoEPluginController: NSObject {
+    var identifier: String
```

Comment:
@RakshithaAcharya Make sure to update the access specifiers in this class too for variables and functions

Reply:
@deepa-iOS-SDK addressed